nntrainer/src/tensor.cpp

```diff
@@ -832,6 +834,7 @@ void Tensor::print(std::ostream &out) const {
     }
     out << "-------" << std::endl;
   }
+  out.copyfmt(init);
```

Comment:
I am not sure about this.
Rest all LGTM

Reply:
In order to use arrange the digit for output, we use setw. But we didn't restore the original. This is for that purpose.